src/kani-compiler/src/parser.rs

```diff
@@ -127,6 +130,11 @@ pub fn parser<'a, 'b>() -> App<'a, 'b> {
                 .long("--assertion-reach-checks")
                 .help("Check the reachability of every assertion."),
         )
+        .arg(
+            Arg::with_name(PRETTY_OUTPUT_FILES)
+                .long("--pretty-files")
```

Comment:
I suggest adding "json" somewhere in the option (e.g. pretty-json-files) to clarify that this is specific for json files.

Reply:
Ok